services/app-web/src/components/FileUpload/FileItem/FileItem.tsx

```diff
@@ -1,141 +1,42 @@
 import React from 'react'
-import { Button } from '@trussworks/react-uswds'
-import classnames from 'classnames'
-import { SPACER_GIF } from '../constants'
+import { FileItemT } from '../FileProcessor/FileProcessor'
 
 import styles from '../FileUpload.module.scss'
+import { Button } from '@trussworks/react-uswds'
+import { SPACER_GIF } from '../constants'
 
-export type FileStatus =
-    | 'DUPLICATE_NAME_ERROR'
-    | 'PENDING'
-    | 'SCANNING'
-    | 'SCANNING_ERROR'
-    | 'UPLOAD_COMPLETE'
-    | 'UPLOAD_ERROR'
-
-export type FileItemT = {
-    id: string
-    name: string
-    file?: File // this becomes undefined after both uploading and scanning has completed with success
-    key?: string // only items uploaded to s3 have this
-    s3URL?: string // only items uploaded to s3 have this
-    status: FileStatus
-}
-
-const DocumentError = ({
-    hasDuplicateNameError,
-    hasScanningError,
-    hasUploadError,
-    hasUnexpectedError,
-}: {
-    hasDuplicateNameError: boolean
-    hasScanningError: boolean
-    hasUploadError: boolean
-    hasUnexpectedError: boolean
-}): React.ReactElement | null => {
-    if (hasDuplicateNameError)
-        return (
-            <>
-                <span className={styles.fileItemBoldMessage}>
-                    Duplicate file
-                </span>
-                <span className={styles.fileItemBoldMessage}>
```

Comment:
Small stylistic things. Not blocking but we can make  follow on tickets to address this sprint

1. We missing conditional logic in documents error to adjust the UI for the the file list versus the table. It looks like in the list, we displayed error text in two spans, whereas in the table we display with a comma between. There could probably be a helper function that uses React.createElement to properly take error text and display it the appropriate way for the renderMode type

2. @joshmfranklin what did we decide about the difference colors of red are we using the same red in the table/list as the alert or no? I think the divergence has been there for awhile
![Screen Shot 2022-01-27 at 9 46 08 AM](https://user-images.githubusercontent.com/10750442/151393455-88a72597-ffa4-4e98-93b2-c16a501a9b95.png)

Reply:
@haworku good catch on the difference in error background color. I don't think I'd noticed them until put the error summary on the page! Lets consolidate them and use a single bg color. Lets match the background of the error message which is `$theme-color-error-lighter` from USWDS and uses the hex of ##f4e3db.